code-challenges.test.js

```diff
@@ -20,33 +20,361 @@ const secretCodeWord2 = "Gobbledygook"
 // Expected output: "G0bbl3dyg00k"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
+const secretCodeWord4 = "EeEe"
+const secretCodeWord5 = "Supercalifragilisticexpialidocious"
+
+// input: a string
+
+// output: a string with the following input letters changed to numbers. "a" to 4, "e" to 3, "i" to 1, and "o" to 0
+
```

Comment:
Surprise! Not Gene but I hope you enjoy my feedback as well. AMAZING pseudocode here!

Reply:
Haha! Of course I enjoy you feedback!!! Thank you!!!